internal/syncer/connector/airtable_test.go

```
@@ -0,0 +1,503 @@
+	}
+}
+
+func TestAirtableValidateConnectorSettingUsesCandidateConfig(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/v0/base/table" || r.URL.Query().Get("pageSize") != "1" {
+			t.Fatalf("unexpected validation URL %s", r.URL.String())
+		}
+		if r.Header.Get("Authorization") != "Bearer request-token" {
+			t.Fatalf("authorization = %q", r.Header.Get("Authorization"))
+		}
+		w.Write([]byte(`{"records":[]}`))
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)